script/__snapshots__/verify-changes.test.ts.snap

```diff
@@ -0,0 +1,849 @@
+@@ -125,3 +88,7 @@ Thanks! 💖
+ 
+ <!-- ALL-CONTRIBUTORS-LIST:END -->
+ <!-- spellchecker: enable -->
++
++<!-- You can remove this notice if you don't want it 🙂 no worries! -->
++
++> 💙 This package was templated with [create-typescript-app](https://github.com/JoshuaKGoldberg/create-typescript-app)."
+`;
+
+exports[`verify bin/index.js 1`] = `""`;
+
+exports[`verify cspell.json 1`] = `
+"diff --git a/cspell.json b/cspell.json
+index 386ca42..494dd2e 100644
+--- a/cspell.json
++++ b/cspell.json
+@@ -1,39 +1,27 @@
+ {
+ 	\\"dictionaries\\": [\\"typescript\\"],
+ 	\\"ignorePaths\\": [
+-		\\"./coverage*\\",
+ 		\\".github\\",
+ 		\\"CHANGELOG.md\\",
++		\\"coverage\\",
+ 		\\"lib\\",
+ 		\\"node_modules\\",
+ 		\\"pnpm-lock.yaml\\"
+ 	],
+ 	\\"words\\": [
+-		\\"allcontributors\\",
+-		\\"apexskier\\",
+-		\\"arethetypeswrong\\",
+ 		\\"Codecov\\",
+ 		\\"codespace\\",
+ 		\\"commitlint\\",
+ 		\\"contributorsrc\\",
+ 		\\"conventionalcommits\\",
+-		\\"execa\\",
+-		\\"infile\\",
+-		\\"joshuakgoldberg\\",
+ 		\\"knip\\",
+ 		\\"lcov\\",
+ 		\\"markdownlintignore\\",
+-		\\"mtfoley\\",
+-		\\"npmignore\\",
+-		\\"npmjs\\",
+ 		\\"npmpackagejsonlintrc\\",
+ 		\\"outro\\",
+ 		\\"packagejson\\",
+ 		\\"quickstart\\",
+-		\\"tada\\",
+ 		\\"tsup\\",
+-		\\"Unstaged\\",
+-		\\"vitest\\",
+ 		\\"wontfix\\"
+ 	]
+ }"
+`;
+
+exports[`verify knip.jsonc 1`] = `
+"diff --git a/knip.jsonc b/knip.jsonc
+index 83c8537..d73a20f 100644
+--- a/knip.jsonc
++++ b/knip.jsonc
+@@ -1,16 +1,6 @@
+ {
+ 	\\"$schema\\": \\"https://unpkg.com/knip@latest/schema.json\\",
+-	\\"entry\\": [
+-		\\"src/index.ts!\\",
+-		\\"src/guide/index.ts\\",
+-		\\"src/initialize/index.ts\\",
+-		\\"src/migrate/index.ts\\",
+-		\\"script/*e2e.js\\"
+-	],
+-	\\"ignoreBinaries\\": [\\"gh\\"],
+-	\\"ignoreExportsUsedInFile\\": {
+-		\\"interface\\": true,
+-		\\"type\\": true
+-	},
+-	\\"project\\": [\\"src/**/*.ts!\\", \\"script/**/*.js\\"]
++	\\"entry\\": [\\"src/index.ts!\\"],
++	\\"ignoreExportsUsedInFile\\": { \\"interface\\": true, \\"type\\": true },
++	\\"project\\": [\\"src/**/*.ts!\\"]
+ }"
+`;
+
+exports[`verify package.json 1`] = `
+"diff --git a/package.json b/package.json
+index 7a0e08a..0c382e9 100644
+--- a/package.json
++++ b/package.json
+@@ -8,14 +8,13 @@
+ 	},
+ 	\\"license\\": \\"MIT\\",
+ 	\\"author\\": {
+-		\\"name\\": \\"Josh Goldberg\\",
++		\\"name\\": \\"Aslemammad\\",
+ 		\\"email\\": \\"npm@joshuakgoldberg.com\\"
+ 	},
+ 	\\"type\\": \\"module\\",
+ 	\\"main\\": \\"./lib/index.js\\",
+ 	\\"bin\\": \\"./bin/index.js\\",
+ 	\\"files\\": [
+-		\\"bin/\\",
+ 		\\"lib/\\",
+ 		\\"package.json\\",
+ 		\\"LICENSE.md\\",
+@@ -24,8 +23,7 @@
+ 	\\"scripts\\": {
+ 		\\"build\\": \\"tsup\\",
+ 		\\"format\\": \\"prettier \\\\\\"**/*\\\\\\" --ignore-unknown\\",
+-		\\"initialize\\": \\"tsx ./src/bin/index.js --mode initialize\\",
+-		\\"lint\\": \\"eslint . .*js --max-warnings 0\\",
++		\\"lint\\": \\"eslint . .*js --max-warnings 0 --report-unused-disable-directives\\",
```

Comment:
[Non-actionable] Aha, a diff I wasn't expecting!

Filed #1089 to track fixing this unexpected difference. No action needed as part of this PR.